### DISCUSSION TITLE
Issue with freshrss data

### DISCUSSION THREAD

**Discussion by rmvpress** [FreshRSS/FreshRSS#6799] [Q&A]
Hello.  I'm hoping for help with my issue regarding my data.  I went to update freshrss via the Linux server guide and it seems I messed up the update.  Luckily I can still see that my data is in /var/lib/docker/volumes/freshrss_data/_data, but when I tried linking this folder to the new freshrss install it seems it isn't loading that data or what have you. I'm hoping there's a way to link the old data to the new install since there's some feeds that I don't remember the addresses of.  If there's a way to simply access the feed URLs so I can manually add them that would work too. My user account is still in the old users directory along with everything else in that _data folder if that helps. Thank you.

**Comment by Alkarex** [comment]
Hello,
I am not sure what you mean with Linux server guide when using Docker, but show the output of `docker volume ls` and `docker ps -a` and your docker-compose or equivalent docker run

**Comment by rmvpress** [comment]
```
DRIVER    VOLUME NAME
local     0f169210e4361b56952da3aee1d4e8a68db65638c233460e98a94fb47871ee97
local     1a6cd73a523b0c786055b680a7900b7975595dfd2f9cea57a2bc8fbb27bd0a8a
local     2d71d75701bd45ab7370211dd5639c14357dd3bba8bf9c24b80630d52f854ace
local     5ad46e7cea67e845c759ea625ef682298fc12173461ea930cf315d72eae99ef3
local     5cb3e2ad7a4b5b467fe73738db5f18c33046bf739e5998a879a1e5dd3ca62488
local     6d4e579e16a71b354a34a2c2f81d0c5035e7670089b58685add0096c01ba56ae
local     9b6f661e2fb458d4bf47060d3a33053e8d4228489722941d71e2aaedd0c615e1
local     72a7475aaf15d4169cf52313c769319430fe09a29e3809a99425839ff1709da3
local     74ab1700709e37ba3424d971a6f5b7e1a1445521db3212bbce91c0be1fada933
local     416ea5a2e1ba96fc490df83716b483e78ce9204d9c7b598c582775951969e790
local     b645e850927d29107ab4e5d054a43ae2eeb8350ae4e0cfbe299c19c69242a41e
local     d599d8912ed2f1a84f28bde8dacdab9f095ba261686049dc1aa76d23f4b99cf2
local     eb5715d18ea0118dd7df5c06cc7931b2d3511a89bbbaa4456699aacd72df9332
local     eed6e93067ee55240633941c20778eb73feb92eb70ecc17587addabf1f5f02da
local     freshrss_data
local     freshrss_extensions
```

```
CONTAINER ID   IMAGE                                   COMMAND            CREATED        STATUS                      PORTS                                        NAMES
a1ca8e280d11   lscr.io/linuxserver/freshrss:latest     "/init"            5 hours ago    Up 5 hours                  0.0.0.0:80->80/tcp, :::80->80/tcp, 443/tcp   freshrss
```

```
docker run -d \
  --name=freshrss \
  -e PUID=1000 \
  -e PGID=1000 \
  -e TZ=America/Phoenix \
  -p 80:80 \
  -v /var/lib/docker/volumes/freshrss_data/_data \
  --restart unless-stopped \
  lscr.io/linuxserver/freshrss:latest
```

also I was following the "updating info" section of this page: https://docs.linuxserver.io/images/docker-freshrss/#application-setup

this string " 1a6cd73a523b0c786055b680a7900b7975595dfd2f9cea57a2bc8fbb27bd0a8a" seems to be the current folder my docker install is using for freshrss 

my old data is here

`/var/lib/docker/volumes/freshrss_data/_data`

**Reply by Alkarex** [reply]
> also I was following the "updating info" section of this page: https://docs.linuxserver.io/images/docker-freshrss/#application-setup

Ah, right, this is not one of our official guides, so I was not familiar with it. I recommend docker-compose instead of manual docker run commands. It looks like you have plenty of anonymous volumes, which is a sign of bad configurations.

Your line `-v /var/lib/docker/volumes/freshrss_data/_data` looks wrong, as it is lacking the right-hand part and also because you are not supposed to manually link to `/var/lib/docker/...`.

Not tested, but after having done a backup, and assuming your old data is in the `freshrss_data` volume, you could try something like:

`-v freshrss_data:/config`

Note that this is for the linuxserver image, which settings (paths) are not compatible with our [official images](https://hub.docker.com/r/freshrss/freshrss/).
